Gemfile

```diff
@@ -8,6 +8,13 @@ group :development do
   gem 'pry'
 end
 
+if RUBY_VERSION < "2.0"
+  gem 'json', '< 2.0.0'
```

Comment:
json should be `~> 1.8.2` IMO, 1.7.x had one of the worst Ruby CVEs https://www.ruby-lang.org/en/news/2013/02/22/json-dos-cve-2013-0269/